### PR TITLE
#203 Redirect back to MyTripsFragment

### LINKED ISSUE #203: Doesn't redirect back on adding a trip
Describe your issue here.

### Steps to reproduce

My Trips -> Add new Trip

### Expected behaviour
Application should redirect back on the list of the Trips 

### Actual behaviour
Display a Toast of successful addition of trip and remain on same activity

### Would you like to work on the issue?
yes

### PR DESCRIPTION
**Pull request by f71uday**
# Description

Added StartActivityForResult in MyTrips Adapter 
Fixes #203

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [ ] `./gradlew checkstyle`

# Checklist:
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Swati4star** [comment]
@f71uday There are merge conflicts. Fix those

**Comment by Swati4star** [comment]
@f71uday I added a new commit in the master branch which removes add trip icon from adapter & has created a new floating action button for the same purpose. You changes are messing up that commit.

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
     private final Context mContext;
     private final List<Trip> mTrips;
     private LayoutInflater mInflater;
-
+    static int ADDNEWTRIP_ACTIVITY_RESULT=203;
```
[the remark this answers is not in the document]
Comment (f71uday) [reply Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java]: Or Any Other Way you can update the list . suggest plaese

[the remark this answers is not in the document]
Comment (Swati4star) [reply Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java]: Where have you made the network call again?

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java [hunk 2]
```diff
@@ -51,6 +51,36 @@ public View getView(final int position, View convertView, @NonNull ViewGroup par
             holder = (ViewHolder) view.getTag();
         }
 
+        if (position == 0) {
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java]: These changes shouldn't be a part of your commit.

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (f71uday) [reply Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java]: I haven't made the network Call yet. As last time you asked me to make changes as my code my making network call again and again. so should I make the calls as i can't see any other way to refresh the list

[the remark this answers is not in the document]
Comment (Swati4star) [reply Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java]: Is the issue resolved by your current code changes?

The problem earlier was, API will be called every time `onResume`. We don't want that. We should make the call only after a trip is successfully sdded

**Comment by Swati4star** [comment]
Do you want to close this one?

**Comment by f71uday** [comment]
yes

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java (hunk 2 above)
Comment (f71uday) [reply Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java]: ok

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (f71uday) [reply Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java]: i got your point

**Pull request closed without merging**